Drop A/B test and search 3 directories deep for tfevent files

joyceerhl opened the pull request:
Based on feedback from today's bug bash: 
* Drop the A/B test between codeactions and codelenses
* Search 3 directories deep for tfevents files instead of 2 directories deep to align with the default directory structure in the output from the starter script at https://pytorch.org/docs/stable/tensorboard.html

Also:
* Use real feature gate name

IanMatthewHuff reviewed (approved):
Code changes look good. But do you still want the code lens clicked telemetry event? Or is that info already captured by another event?

joyceerhl commented:
@IanMatthewHuff, in the command handler we capture whether the command was invoked from the prompt, codelens or palette. Those two events I removed are redundant events that I added at Jeffrey's request. Good question though, went back to verify that I was creating the codelens with the right args and noticed some dead code that I could eliminate.

karrtikr commented on src/client/tensorBoard/tensorBoardFileWatcher.ts:
```diff
@@ -15,9 +15,7 @@ import { TensorBoardPrompt } from './tensorBoardPrompt';
-    private globPattern1 = '*tfevents*';
-
-    private globPattern2 = '*/*tfevents*';
+    private globPatterns = ['*tfevents*', '*/*tfevents*', '*/*/*tfevents*'];
```
I would prefer if the file watchers are put behind some sort of flag, whether a experiment flag or some settings flag. But if this has been well discussed and thought through I'm fine with it.

joyceerhl replied:
The file watchers are not created unless the user is in the experiment:

https://github.com/microsoft/vscode-python/blob/4d539ba41f88edb49aed348857768755872195fa/src/client/tensorBoard/tensorBoardFileWatcher.ts#L34

Is that what you're referring to? Agreed that file watchers should not be running unless the user is in the experiment.

karrtikr replied:
Yes, thanks!

karrtikr reviewed (approved):
LGTM otherwise

ericsnowcurrently reviewed (approved):
LGTM
